lib/rucio/web/ui/static/did.js

```diff
@@ -851,22 +851,6 @@ build_page = function() {
             $('#loading').html('<font color="red">Could not find the DID.</font>');
         }
     });
-
```

Comment:
not necessary to remove the function, is it?

Reply:
shall I just comment it out for later use?

Reply:
just leave it in. you removed the call in the other file

Reply:
But the function is called on page load, I guess. Then the error message would be still visible.

Reply:
The function can stay; I think what we should comment out is the error part (line 866-868) - so that the error is not displayed even if the did is there.

Reply:
ok

Reply:
I wonder though if we skip the error, if the table is still rendered, even if there is no data? (Maybe @tbeerman knows?)

Reply:
Then the html can stay, too, I guess.